docs/docs/workflows/sharing/README.md

```diff
@@ -0,0 +1,109 @@
+
+::: tip Connected Accounts are not shared
+
+Shared workflow links **will not** include your own connected accounts. Instead in this new workflow the user of your workflow link will be prompted to connect their own accounts.
+
+:::
+
+## Versioning
+
+When you create a shared link for your workflow, that link is frozen to the version of your workflow at the time the link was created.
+
+If changes are made to the original workflow, the changes will _not_ be included in the shared workflow link, nor in any workflows copied from the original shared link.
+
+[Generate a new share link](#creating-a-share-link-for-a-workflow) to include new changes to a workflow.
+
+::: tip Share Links persist
+
+You can create multiple share links for the same workflow with different prop configurations, or even different steps.
+
+Share links will not expire or be overridden.
+
+:::
+
+## Frequently Asked Questions
+
+### If changes are made to the original workflow, will copied versions of the workflow from the shared link also change?
+
+No, workflows copied from a shared link will have separate version histories from the original workflow. You can modify your original workflow and it will not affect copied workflows.
+
+### Will my connected accounts be shared with the workflow?
+
+No, your connected accounts **will not** be shared. Instead copied workflows will display a slot in actions that required a connected account, so the user of the copied workflow can provide their own accounts instead.
+
+For example, if one of your steps relies on a Slack connected account to send a message, then the copied workflow will display the need to connect a Slack account.
+
+### I haven't made any changes to my workflow, but if I generate another shared link will it override my original link?
+
+No, if the steps and prop configuration of the workflow is exactly the same, then the shared link URL will also be exactly the same.
+
+The shared workflow link is determined by the configuration of your workflow, it's not a randomly generated ID.
+
+### Will generating new shared links disable or delete old links?
+
+No, each link you generate will be available even if you create new versions based on changes or included props from the original workflow.
+
+### What plan is this feature available on?
+
+Sharing workflows via link is available on all plans, including the Free tier plans.
+
+### Do users of my workflow need to have a subscription?
+
+To copy a workflow, a subscription is not required. But the copied workflow is subject to the current workspace's plan limits.
+
+For example, if a workflow requires more connected accounts than what's available on the [Free tier](/pricing/#free-tier), then users of your workflow will require a plan to run the workflow properly.
```

Comment:
```suggestion
For example, if a workflow requires more connected accounts than what's available on the [free tier](/pricing/#free-tier), then users of your workflow require a plan to run the workflow properly.
```

Reply:
Changed "Free tier" to "free tier" to follow Google's style guide on capitalization.